en/docs/references/ballerina-expression-syntax.md

```diff
@@ -312,111 +312,110 @@ The following operators are available in the expression editor for numeric value
 
 ### Reading a header value in the request
-A typical HTTP request contains many headers.  These include standard and custom headers. You can use the `request` variable's `getHeader` function to read a specific header value in any expression-enabled input field.
+A typical HTTP request contains many headers.  These include standard and custom headers. You can use the request variable's `getHeader` function to read a specific header value in any expression-enabled input field.
 
 #### Examples
 
-- Reading the value of an HTTP header (`Access-Control-Allow-Origin`) passing the header name.
+- Reads the value of an HTTP header on passing the header name (`Access-Control-Allow-Origin`):
 
   ```ballerina
   check request.getHeader("Access-Control-Allow-Origin")
   
   ```
   
-- Reading the value of an HTTP header by passing a variable that contains the header name (`headerName`). Note the omission of the double-quotes.
+- Reads the value of an HTTP header on passing a variable that contains the header name (`headerName`). Note the omission of the double quotes:
 
   ```ballerina
   check request.getHeader(headerName)
   ```
 
 ### Checking whether a header is present in the request
 
-Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  function in the `request` object for this purpose.
+Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the `hasHeader`  function in the request object for this purpose.
 
 #### Examples
 
-- Checking if a particular HTTP header (`content-length`) exists given the header name.
+- Checks if a particular HTTP header (`content-length`) exists, given the header name:
 
   ```ballerina
   request.hasHeader("content-length")
   ```
   
-- Checking if a particular HTTP header (`content-length`) exists by passing a variable (`contentLength`) containing the header name. Note the omission of the double-quotes.
+- Checks if a particular HTTP header (`content-length`) exists by passing a variable (`contentLength`) containing the header name. Note the omission of the double quotes:
 
   ```ballerina
   request.hasHeader(contentLength)
   ```
 
-### Reading the JSON Payload of the request
+### Reading the JSON payload of the request
 
-JSON is a common content type used for HTTP communication. The `request` variable’s `getJsonPayload` function can be used to read the JSON payload sent with the HTTP request. Note that the `getJsonPayload` function’s return type is `json|ClientError`.
+JSON is a common content type used for HTTP communication. You can use the request variable’s `getJsonPayload` function to read the JSON payload sent with the HTTP request. Note that the `getJsonPayload` function’s return type is `json|ClientError`.
```

Comment:
Why are we removing backtick from `request`?

Reply:
Check all the places.

Reply:
Removed because in such occurrences `request` is not really the variable name. Does it make sense to use backticks in such occurrences?

Reply:
Formatted appropriate occurrences of the term request with backticks.